File: templates/python/storage_battery_polar_signature.py
"""storage_battery_polar_signature: 储能与电池极坐标指纹（polar 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='storage and battery analysis: polar signature'):
    return make_template_figure('polar_profile', seed=2410, title=title, domain='storage and battery analysis', topic='polar signature')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/storage_battery_polar_signature.m
function fig = storage_battery_polar_signature()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('polar_profile', 2410, 'storage and battery analysis: polar signature', 'storage and battery analysis', 'polar signature');
end
